styleguide/template.md

```diff
@@ -409,18 +413,31 @@ It's important to use the following alert styles so they render with the proper
 And they'll render like this:
 ![Alert styles](../images/alerts.png)
 
-###  Includes
+### Includes
 
 You can embed the Markdown of one file into another using an include.
 
 [!INCLUDE[sample include file](../includes/sampleinclude.md)]
 
+### Checked lists
+
+A custom style is available for bulleted lists. You can render lists with check marks.
```

Comment:
A custom style is available for bulleted lists, or a custom style is available for checked lists? (Or should it be just check lists?).

Reply:
I just copied and paste from our guidance and didn't even pay attention. I think it should be checked lists.